File: testes/py/1/2.py
# Codigo exemplo do tutorial de opencv python do site oficial:
# https://docs.opencv.org/trunk/dc/d2e/tutorial_py_image_display.html

import numpy as np
import cv2 as cv
img = cv.imread('../../input/img/lena.jpg',0)
cv.imshow('image',img)
k = cv.waitKey(0)
if k == 27:         # wait for ESC key to exit
	cv.destroyAllWindows()
elif k == ord('s'): # wait for 's' key to save and exit
	cv.imwrite('../../output/img/lena_gray.png',img)
	cv.destroyAllWindows()

File: testes/cpp/1/2.cpp
#include <opencv2/opencv.hpp>
#include <iostream>

int main(int argc, char** argv)
{
	// Read the image file
	cv::Mat image = cv::imread("../../input/img/lena.jpg");

	// Check for failure
	if (image.empty()) 
	{
		std::cout << "Could not open or find the image. Press some key" << std::endl;
		std::cin.get(); //wait for any key press
		return -1;
	}

	std::cout << "Do not close the window through the window's X." << std::endl;
	std::cout << "Press some key to close it." << std::endl;

	cv::String windowName = "Lena"; //Name of the window
	cv::namedWindow(windowName); // Create a window
	cv::imshow(windowName, image); // Show our image inside the created window.
	cv::waitKey(0); // Wait for any keystroke in the window.
					// O valor do argumento é a quantidade de milisegundos para esperar por alguma resposta
	cv::destroyWindow(windowName); //destroy the created window

	return 0;
}